beat-editor/src/styles.scss

```diff
@@ -120,8 +120,178 @@ button {
   font-family: 'Poppins', sans-serif !important;
 }
 
+.shortcut-button {
+  background-color: #47555e;
```

Comment:
I do see that this hex color is used a lot here. Usually with LESS and SASS, we would make a global variable with this hex color or any other type of styling that we see used a lot and call it here so it's easier to read. Since we're in the midst of migrating everything over to Tailwind, we can keep this here.